ErenAkpinar/hw4/foodbasket-web/index.html

```diff
@@ -0,0 +1,185 @@
+                    <p ng-if="loading">Loading...</p>
+                    <ul class="list-group">
+                        <li ng-repeat="restaurant in restaurants track by $index"
+                            class="list-group-item justify-content-between">
+                            <a href="#" ng-click="getRestaurantFoods(restaurant.id)">
+                                {{$index + 1}}. {{restaurant.name}} ({{restaurant.district}})
+                            </a>
+                            <a href="">
+                                <span class="badge badge-success badge-pill">
+                                     <i class="fa fa-hand-o-right"></i>
+                               </span>
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="card mb-4" ng-if="restaurantFoods">
+                <div class="card-block">
+                    <h2 class="card-title" ng-bind="restaurantFoods.name + ' (' + restaurantFoods.district + ')'"></h2>
+                    <p class="card-text">
+                        <ul class="list-group">
+                            <li ng-repeat="food in restaurantFoods.foods track by $index"
+                                class="list-group-item justify-content-between">
+                                <div>
+                                    <i class="fa fa-shopping-cart "></i>
+                                    {{$index + 1}}. {{food.name}} - ${{food.price}}
+                                </div>
+                                <div ng-controller="OrderController">
+                                    <a href="#" ng-click="addOrderItem(food.id, food.price)">
+                                       <span class="badge badge-success badge-pill">
+                                             <i class="fa fa-plus-circle"></i>
+                                       </span>
+                                    </a>
+                                </div>
+
+                            </li>
+                        </ul>
+                    </p>
+                </div>
+                <div class="card-footer text-muted">
+                    <a href="#" ng-bind="restaurantFoods.district"></a>
+                    <div class="pull-right">
+                        <i class="fa fa-phone "></i> <span ng-bind="restaurantFoods.phoneNumber"></span>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<footer class="py-3 bg-inverse fixed-bottom">
+    <div class="container">
+        <p class="m-0 text-center text-white">Copyright &copy; Food Basket 2017 <a href="htpp://erenakpinar.com">ea</a>
+        </p>
+    </div>
+</footer>
+
+<script type="text/javascript" src="services/dataService.js"></script>
+
+<script type="text/javascript" src="lib/angular.min.js"></script>
+<script type="text/javascript" src="lib/angular-route.min.js"></script>
```

Comment:
bu route'u attığınız örnekte kullandığınız için koymuştum Gizem abla ancak kontrolünü yapmamıştım.